.github/workflows/components-bump-version-release.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           scope: "@mapsindoors"
       # Build the project to ensure it's up to date. If this steps fails, the job fails.
       - name: Build
-        run: cd packages/components && npm ci && npm run build
+        run: npm ci && npx lerna run build && cd packages/components && npm run build
```

Comment:
Is everything necessary? `npx lerna run build` will run the `npm run build` of all packages.

Reply:
@ammapspeople What would you suggest it should be then? This? ⤵️

```suggestion
        run: npm ci && npx lerna run build
```

Reply:
I like that better, since that seems to make sure than any changes made to MIDT will also be included when building the components.